src/routes/transactions/mappers/common/human-descriptions.mapper.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { encodeFunctionData, parseAbi } from 'viem';
+import { faker } from '@faker-js/faker';
+import { AddressInfo } from '../../../common/entities/address-info.entity';
+import { HumanDescriptionMapper } from './human-description.mapper';
+import { TokenRepository } from '../../../../domain/tokens/token.repository';
+import { tokenBuilder } from '../../../../domain/tokens/__tests__/token.builder';
+import { ILoggingService } from '../../../../logging/logging.interface';
+
+import Messages from '../../../../datasources/human-description-api/json';
+import { HumanDescriptionApi } from '../../../../datasources/human-description-api/human-description-api.service';
+import { MAX_UINT256 } from '../../constants';
+import { SafeAppInfo } from '../../entities/safe-app-info.entity';
+
+const tokenRepository = jest.mocked({
+  getToken: jest.fn(),
+} as unknown as TokenRepository);
+
+const mockLoggingService = {
+  info: jest.fn(),
+  debug: jest.fn(),
+  error: jest.fn(),
+  warn: jest.fn(),
+} as unknown as ILoggingService;
+
+const humanDescriptionAPI = new HumanDescriptionApi(Messages);
+const toAddress = new AddressInfo(faker.finance.ethereumAddress());
+const chainId = faker.string.numeric();
+const token = tokenBuilder()
+  .with('decimals', 18)
+  .with('name', 'Test Token')
+  .with('symbol', 'TST')
+  .build();
+const abi = parseAbi(['function transfer(address, uint256)']);
+const mockTransferData = encodeFunctionData({
+  abi,
+  functionName: 'transfer',
+  args: [
+    '0x7a9af6Ef9197041A5841e84cB27873bEBd3486E2',
+    BigInt('21000000000000000000'),
```

Comment:
Let's generate a random value for this as well.

```suggestion
    `faker.number.bigInt()`,
```

Reply:
done